pricegraph/src/orderbook/order.rs

```diff
@@ -63,6 +61,17 @@ impl OrderMap {
         })
     }
 
+    /// Returns an iterator over the orders matching a given sell token.
+    pub fn pairs_for_sell_token(
```

Comment:
nit: maybe integrate the fact that we return orders in the name? `pairs_and_orders_with_sell_token` or something like that?